app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+
+
+    var watch = function(context, callback, opts ) {
+      return DataService.watch(mobileclientVersion, context, callback, opts);
+    };
+
+    var getMobileClients = function(namespace) {
+      return DataService.list(mobileclientVersion, {namespace: namespace})
+        .then(function(clients) {
+          return clients.by("metadata.name");
+        });
+    };
+
+    var filterExcluded = function(serviceInstance, mobileClients) {
```

Comment:
Duplication of CRUDL functionality, same as  in `MobileServicesService`. Should be abstracted to `MobileCommonService` and then injected into `MobileClientsService` and `MobileServicesService`

Reply:
Ok, but won't that just add extra overhead?

Reply:
@dimitraz DRY and KISS principles should be used. COPY principle create extra overhead :))